cfme/tests/automate/test_git_import.py

```diff
@@ -17,11 +17,19 @@
     pytest.mark.meta(server_roles="+git_owner"),
 ]
 
-GIT_REPO_URL = cfme_data.automate_links.datastore_repositories.manageiq_automate
+GIT_REPO_URL = None
+
+try:
+    GIT_REPO_URL = cfme_data.automate_links.datastore_repositories.manageiq_automate
+except AttributeError:
+    pass
 
 
 @pytest.fixture
 def imported_domain(appliance):
+    if GIT_REPO_URL is None:
```

Comment:
Could also do this via a separate fixture instead of having to re-use the same block?